src/core/core.js

```diff
@@ -815,4 +824,11 @@ export default class Core {
     }
     return null;
   }
+
+  _getMergedClientSdk () {
+    return {
+      ...this._customClientSdk,
+      ANSWERS_SEARCH_UI_SDK: LIB_VERSION
```

Comment:
to confirm my own understanding, we are getting this LIB_VERSION during the build process by checking git tag?

Reply:
yup!